src/geo/transform.js

```diff
@@ -1262,6 +1271,20 @@ class Transform {
         return posMatrix;
     }
 
+    calculateDistanceTileMatrix(unwrappedTileID: UnwrappedTileID): Float32Array {
+        const fogTileMatrixKey = unwrappedTileID.key;
+        const cache = this._distanceMatrixCache;
+        if (cache[fogTileMatrixKey]) {
+            return cache[fogTileMatrixKey];
+        }
+
+        const posMatrix = this.calculatePosMatrix(unwrappedTileID, this.cameraWorldSize);
+        mat4.multiply(posMatrix, this.centerDistanceMatrix, posMatrix);
+
+        cache[fogTileMatrixKey] = new Float32Array(posMatrix);
+        return cache[fogTileMatrixKey];
```

Comment:
Is that a leftover from switching to having a different matrix from fog? Should it be the following:
```suggestion
        const matrixKey = unwrappedTileID.key;
        const cache = this._distanceMatrixCache;
        if (cache[matrixKey]) {
            return cache[matrixKey];
        }

        const posMatrix = this.calculatePosMatrix(unwrappedTileID, this.cameraWorldSize);
        mat4.multiply(posMatrix, this.centerDistanceMatrix, posMatrix);

        cache[matrixKey] = new Float32Array(posMatrix);
        return cache[matrixKey];
```